prow/jobs/kyma-project/compass-manager/compass-manager-tests.yaml

```diff
@@ -0,0 +1,34 @@
+presubmits: # runs on PRs
+  kyma-project/compass-manager:
+    - name: pre-main-compass-manager-tests
+      annotations:
+        pipeline.test: "fast-integration"
+        pipeline.trigger: "pr-submit"
+        pipeline.type: "integration"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pre-main-compass-manager-tests"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+      optional: false
+      run_if_changed: '^(go.mod|go.sum)$|^*/(.*.go|Makefile|Dockerfile|.*.sh)'
+      skip_report: false
+      decorate: true
+      path_alias: github.com/kyma-project/compass-manager
+      cluster: untrusted-workload
+      max_concurrency: 10
+      branches:
+        - ^main$
+      spec:
+        containers:
+          - image: "europe-docker.pkg.dev/kyma-project/dev/test-infra/buildpack-golang:PR-7292"
+            command:
+              - "/bin/sh"
+            args:
+              - "-c"
+              - "make test"
```

Comment:
We _could consider_ using a script in our repo here, which would make it easier to modify the pipeline if we ever need anything more advanced (like displaying coverage data)

Reply:
I think is not necessary to create a separate script for testing. We can easily modify the target in the Makefile.

Reply:
But with my coverage example - when running locally it makes sense to just use the coverage file, but in pipelines it makes more sense to display it directly after tests, with no user interaction.